src/components/LanguageButton.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useCallback } from "react";
+import styled from "styled-components";
+import {
+  space,
+  typography,
+  layout,
+  flexbox,
+  SpaceProps,
+  TypographyProps,
+  LayoutProps,
+  FlexboxProps,
+} from "styled-system";
+import i18next from "i18next";
+
+const Container = styled.div<FlexboxProps & LayoutProps>`
+  ${flexbox};
+  ${layout};
+`;
+
+const LangButton = styled.button<SpaceProps & TypographyProps>`
+  background: transparent;
+  border: none;
+  ${typography};
+  ${space};
+  transition: transform 0.2s;
+  &:hover {
+    transform: scale(1.05);
+   
+`;
+
+const LanguageButton = () => {
```

Comment:
i feel like this component/file should be called LanguageButtons (plural) since there are more than one button

Reply:
NB. im approving but you still need to make the change (its a way of saying i trust you to make the change without another check)